config.rb

```diff
@@ -25,7 +25,26 @@
   blog.per_page = 10
 end
 
-page "/blog/feed.xml", layout: false
+# middleman-blog does not allow the `mount_at_root` like feature
+# so we define two blog instances for English one and localized one.
+
+# English blog at /blog
+activate :blog do |blog|
+  blog.name = 'en'
+  # This will add a prefix to all links, template references and source paths
+  blog.prefix = "blog"
+  configure_blog(blog)
+end
+
+# Localized blogs at /{lang}/blog
+activate :blog do |blog|
+  blog.name = 'i18n'
```

Comment:
Hmm.  I didn't realize that.  Since we can't use proper HTTP redirects my preference is to keep the English blog available at it's current URLs and add `/blog/ja` for Japenese blog posts but not `/blog/en` as a mirror.  Is that feasible?